weaviate/values.yaml

```diff
@@ -13,6 +13,53 @@ image:
   pullPolicy: IfNotPresent
   pullSecrets: []
 
+experimental:
+  querier:
+    enabled: false
+    replicas: 3
+    image: semitechnologies/weaviate-experimental:preview-chore-change-default-configs-on-querier-23d39eb
```

Comment:
can we follow the same pattern for defining the Weaviate image as we do it in the Weaviate STS:
<img width="315" alt="Screenshot 2024-10-17 at 15 33 27" src="https://github.com/user-attachments/assets/e28f34eb-33c0-4e46-95bf-21c2118d8db4">